### DISCUSSION TITLE
Feature request: Update Pocketbase to latest version

### DISCUSSION THREAD

**Discussion by vtisnado** [pockethost/pockethost#212] [Q&A]
Hi @benallfree It will be nice to have an option to upgrade the Pocketbase version. 

Currently I have the **v0.12.3**, but the latest version is **v0.13.4**.

**Comment by benallfree** [comment]
Hey @vtisnado, Yes I'll make 0.13.x available. Once I make the update, all newly created instances will run 0.13.x instead of 0.12.x

There is currently an intentional manual migration process in place until PocketBase stabilizes its upgrade process. #179 discusses this a little bit, as does #72.

If you need an existing instance upgraded, please contact me.

**Comment by vtisnado** [comment]
Oh now I remember I saw #179, I was looking for that thread actually, but could not found it.

Oh thanks, maybe you can upgrade my instance `crashcourses` if is to many trouble, I can destroy it and re-create it again :)
